src/BenchmarkDotNet.Core/Extensions/CommonExtensions.cs

```diff
@@ -60,6 +61,30 @@ public static string ToStr(this double value, string format = "0.##")
             return string.Format(HostEnvironmentInfo.MainCultureInfo, $"{{0:{format}}}", args);
         }
 
+        /// <summary>
+        /// Gets column title formatted using the specified style
+        /// </summary>
+        public static string GetColumnTitle(this IColumn column, ISummaryStyle style)
+        {
+            if (!style.PrintUnitsInHeader)
+            {
```

Comment:
In BenchmarkDotNet, we usually don't use brackets for simple one-line `if` statements.